wcfsetup/install/files/lib/util/HeaderUtil.class.php

```diff
@@ -47,8 +47,15 @@ public static function setCookie($name, $value = '', $expire = 0) {
 		if ($addDomain && strpos($cookieDomain, ':') !== false) {
 			$cookieDomain = explode(':', $cookieDomain, 2)[0];
 		}
+	    
+	    	$sameSitePolicy = "";
```

Comment:
This value is immediately reassigned.

Reply:
Well, it's only reassigned if $sameSiteOption is not 'none'. Otherwise it never gets assigned, and I believe it is a good practice to always declare variables that are used.